test/orange/univ/detail/SenseCalculator.test.cc

```diff
@@ -3,7 +3,7 @@
 // See the top-level COPYRIGHT file for details.
 // SPDX-License-Identifier: (Apache-2.0 OR MIT)
 //---------------------------------------------------------------------------//
-//! \file orange/universes/detail/SenseCalculator.test.cc
+//! \file orange/univ/detail/SenseCalculator.test.cc
```

Comment:
What is the advantage (besides saving 5 characters) of using `univ` vs `universe(s)`?

Reply:
Just keeping the paths short—I'd rather type `alg` than `algorithm`, `univ` than `universes`; but I've also left longer ones that might have ambiguous interpretations (no `dist`, `const`, etc). I'm happy to put it to a vote among the team :)